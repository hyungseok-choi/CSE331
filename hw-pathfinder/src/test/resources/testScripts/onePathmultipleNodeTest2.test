CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C

AddEdge simpleGraph A B 10.0
AddEdge simpleGraph A C 1.0

FindPath simpleGraph A B
