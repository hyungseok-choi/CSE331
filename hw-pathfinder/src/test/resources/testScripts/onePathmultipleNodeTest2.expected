created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph

added edge 10.000 from A to B in simpleGraph
added edge 1.000 from A to C in simpleGraph

path from A to B:
A to B with weight 10.000
total cost: 10.000
